### PR TITLE
Adjust queueing and batching configurarion for OTC according to testi…

### PR DESCRIPTION
**Pull request by pdelewski**
…ng results

##### Description

Fill in your description here.

---

##### Checklist

- [ ] Changelog updated

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -3977,7 +3977,7 @@ metadata:
         ## ref: https://github.com/SumoLogic/opentelemetry-collector/blob/main/processor/batchprocessor
         batch:
           ## Number of spans after which a batch will be sent regardless of time
-          send_batch_size: 16_384
+          send_batch_size: 1_024
           ## Time duration after which a batch will be sent regardless of size
           timeout: 30s
```
Comment (pdelewski) [inline deploy/helm/sumologic/values.yaml:3982]: Should we update timeout as well?

Comment (swiatekm-sumo) [reply deploy/helm/sumologic/values.yaml:3982]: Yeah, a lower value like `1s` is better imo.

**Review by swiatekm-sumo — APPROVED** [review]
LGTM, do you plan on doing logs in a separate PR?

**Comment by pdelewski** [comment]
> LGTM, do you plan on doing logs in a separate PR?

From what I see, originally it was done only for metrics, so I updated only metrics. I can prepare separate PR for logs

File: deploy/helm/sumologic/values.yaml (hunk 1 above)
Comment (pdelewski) [reply deploy/helm/sumologic/values.yaml:3982]: I will update that

**Pull request merged**